dotCMS/src/main/java/com/dotmarketing/portlets/templates/business/TemplateAPI.java

```diff
@@ -16,6 +16,9 @@
 
 public interface TemplateAPI {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Move constants to a class or enum. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1214)